### ISSUE TITLE
TXT exports lack reply support

### ISSUE THREAD

**Issue by notevenaperson** [Tyrrrz/DiscordChatExporter#1118]
### Version

2.40

### Flavor

GUI (Graphical User Interface), CLI (Command-Line Interface)

### Platform

All

### Export format

TXT

### Steps to reproduce

.

### Details

- Expected behavior: TXT export would represent when a message is in direct reply to another. 
- Actual behavior: A message in reply to another appears like a standalone message, making it hard to follow conversations in the archive.

Being able to reference past messages in reply was added to discord fairly recently, relatively to the service's age, but nonetheless conversations have come to rely on bringing older messages back in replies. DiscordChatExporter's HTML archive has been updated to support replies, indicating to me that you'd agree on its importance. 

I suggest that in the TXT export, since we can't do links, you include a short quotation of the message being referenced and leave it at that.

### Checklist

- [X] I have looked through existing open and closed issues to make sure that this bug has not been reported before
- [X] I have provided a descriptive title for this issue
- [X] I have made sure that that this bug is reproducible on the latest version of the application
- [X] I have provided all the information needed to reproduce this bug as efficiently as possible
- [ ] I have sponsored this project

**Comment by Tyrrrz** [comment]
This is not a bug. Export formats were never meant to have feature parity. The plain text format made more sense in 2017, when I added it, because Discord chatting features were pretty basic back then. Now, it's not a format you should be using (same as with CSV). I'm not planning to evolve it further. If you want rendering capabilities beyond just plain text, you should be exporting in HTML instead.

**Comment by notevenaperson** [comment]
Okay. You don't have to work on it if you don't want to. I'll opine that representing replies to make the conversation minimally readable certainly doesn't special rendering capabilities. It just needs a short quotation of the message being referenced.

```
[2023-08-14 10:15:30Z] person1
sunt repellat alias odio ut. facilis labore possimus qui dolor minus. possimus at aut eligendi architecto est molestiae voluptas aut. quam ipsam rerum labore dolorem expedita sed.

...

[2023-08-14 11:46:40Z] person2
> sunt repellat alias odio ut. facilis labore possimus...
did you post lorem ipsum for no reason or are you actually learning latin
```
